README.rst

```diff
@@ -5,6 +5,15 @@ Scrapyrt (Scrapy realtime)
 .. image:: https://travis-ci.org/scrapinghub/scrapyrt.svg?branch=master
     :target: https://travis-ci.org/scrapinghub/scrapyrt
 
+.. image:: https://img.shields.io/pypi/pyversions/scrapyrt.svg
+    :target: https://pypi.python.org/pypi/scrapyrt
+
+.. image:: https://img.shields.io/pypi/v/scrapyrt.svg
+    :target: https://pypi.python.org/pypi/scrapyrt
+
+.. image:: https://img.shields.io/pypi/l/scrapyrt.svg
+    :target: https://pypi.python.org/pypi/scrapyrt
+
```

Comment:
this looks like this https://github.com/pawelmhm/scrapyrt/blob/6adc7ce91f70d65ecce9101bab12cfea1c4c5298/README.rst much better imo